module.json

```diff
@@ -44,6 +44,9 @@
     },
     "arm-ssg": {
       "mbed-hal-arm-ssg": "^1.0.0"
+    },
+    "atmel": {
+      "mbed-hal-atmel": "^0.1.0"
```

Comment:
@0xc0170  Referred http://yottadocs.mbed.com/reference/module.html
I am seeing different behaviour . The following dependencies downloads mbed-hal-atmel 0.1.0 but latest version available is "0.1.1". If I change it as ">0.1.0" it gets 0.1.1.  Any idea ?

atmel": {
      "mbed-hal-atmel": "^0.1.0"
}